docs/src/__examples__/ButtonLink/DEFAULT.tsx

```diff
@@ -39,7 +39,7 @@ export default {
           name: "type",
           type: "select",
           defaultValue: "primary",
-          options: ["primary", "secondary", "critical", "primarySubtle", "criticalSubtle", "white"],
+          options: ["primary", "secondary", "critical"],
```

Comment:
I'm not sure about that, because in Figma it does not have `dark` colors, so it would be much better to relay on figma designs. If it's not changed there yet, we have to ask them to change and implement after.

Reply:
Sure 👍